src/types.ts

```diff
@@ -1,9 +1,11 @@
 import type Func from './ast/function';
 import type Node from './ast/node';
 import type Var from './ast/variable';
+import type UnsafeRaw from './raw';
 import type Tag from './tag';
+import type Token from 'markdown-it/lib/token';
 
-export type { Node, Tag };
+export type { Node, Tag, Token };
```

Comment:
@rpaul-stripe are we sure we want to expose `Token` here? Just want to be safe around over-coupling us to `markdown-it`

Reply:
While it's convenient since you're bundling markdown-it internally, that seems like a valid point, I don't have a problem with using a direct dependency on @types/markdown-it instead.